app/controllers/api/v0/manifests_controller.rb

```diff
@@ -25,26 +25,18 @@ def create
 
   def show
     begin
-      if params[:id]
-        manifest = Manifest.find(params[:id])
-      elsif params[:tracking_number]
-        manifest = Manifest.find_by!("content -> 'generator' ->> 'manifest_tracking_number' = ?", params[:tracking_number])
-      end
+      manifest = find_manifest
     rescue ActiveRecord::RecordNotFound => _error
       render json: {}, status: 404
       return
     end
 
-    render json: manifest.to_json
+    render json: manifest.to_public_json
```

Comment:
I had that in the PR that was closed, but the serializer did make it into the Rails PR :)